.travis.yml

```diff
@@ -6,23 +6,33 @@ cache:
   directories:
     - vendor
     - $HOME/.composer/cache
-    - /tmp/Elcodi/
 
 php:
   - 5.4
   - 5.5
   - 5.6
-  - hhvm
   - 7.0
+  - hhvm
 
 env:
   global:
     - secure: b4fh7VMvVktVPsMY9Z8/YOHXWoSGrl/caETmnYsuF3dTV3eOLIQxtP9eW58JBpH14CBLxzbs8D7l2bcvHShsU946Pgxn58xAWO4QLSiZtEJGOB/bVu2s7V/ckEmzbCc83TRiDb4EBaMWLoFWRQn87m00zky4URtDjMGjGgdAfOA=
 
+matrix:
+  include:
+    - php: 5.4
+      env: SYMFONY_VERSION="~2.7.0"
+    - php: 5.6
+      env: SYMFONY_VERSION="~2.8.0"
+    - php: 7.0
+      env: SYMFONY_VERSION="^3.0.0"
```

Comment:
Until we are not ready for Symfony 3, please add this into the `allow_failures` list.